packages/expo-notifications/android/src/main/java/expo/modules/notifications/notifications/model/NotificationAction.java

```diff
@@ -7,6 +7,9 @@
 
 /**
  * A class representing a single notification action button.
+ *
+ * TODO vonovak: no need to implement serializable, parcelable is enough for storing
```

Comment:
I had thought that actions were serialized and sent to JS as part of events, so maybe Serializable is needed here?

Reply:
Hello, the response to a button press is sent to JS via [here](https://github.com/expo/expo/blob/6845f2657d4deece98f925b7165f49944f8016d7/packages/expo-notifications/android/src/main/java/expo/modules/notifications/notifications/NotificationSerializer.java#L42-L42)

and the actions are passed from JS to native via [here](https://github.com/expo/expo/blob/6845f2657d4deece98f925b7165f49944f8016d7/packages/expo-notifications/android/src/main/java/expo/modules/notifications/notifications/categories/ExpoNotificationCategoriesModule.kt#L98-L98)

I might be wrong but I'm almost certain there's no need to implement the Serializable interface.